core/jobs/batch_jobs/exp_version_history_computation_job.py

```diff
@@ -91,7 +115,7 @@ def generate_exploration_from_snapshot(
                 snapshot_dict = snapshot_model.content
                 exp_id = snapshot_model.get_unversioned_instance_id()
                 model_class = exp_models.ExplorationModel
-                reconstituted_model = model_class(id=exp_id)._reconstitute(  # pylint: disable=protected-access
+                reconstituted_model = model_class(id=exp_id)._reconstitute( # pylint: disable=protected-access
```

Comment:
```suggestion
                reconstituted_model = model_class(id=exp_id)._reconstitute(  # pylint: disable=protected-access
```
please do not remove this space.

Reply:
Done.